data/ideas.json

```
@@ -185,6 +185,12 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     },
+    {
+        "title": "Picture Finder",
+        "description": "A website where we search for any picture that is available on unsplash. we can use unsplash API for this, also we can use axios for fetching the data and a UI library for styling.",
+        "level": "beginner",
+        "tags": "api, react"
```

- The first tag is used to match projects with an idea, so please add the book-app tag
- Remove the programming tag, due to is only an idea so the everyone could make on whatever programming language
```suggestion
        "tags": "picture-finder, api"
```